src/plugins/data_views/public/data_views/data_views_api_client.ts

```diff
@@ -6,33 +6,66 @@
  * Side Public License, v 1.
  */
 
-import { HttpSetup } from '@kbn/core/public';
+import { HttpSetup, HttpResponse } from '@kbn/core/public';
 import { DataViewMissingIndices } from '../../common/lib';
 import { GetFieldsOptions, IDataViewsApiClient } from '../../common';
 import { FieldsForWildcardResponse } from '../../common/types';
-import { FIELDS_FOR_WILDCARD_PATH } from '../../common/constants';
+import { FIELDS_FOR_WILDCARD_PATH, FIELDS_PATH } from '../../common/constants';
 
 const API_BASE_URL: string = `/api/index_patterns/`;
 const version = '1';
 
+async function sha1(str: string) {
```

Comment:
If it needs a stable hash function, there's also the `import { stringHash } from '@kbn/ml-string-hash';` available

Reply:
@dej611 Its unclear to me why I would chose that over `sha1`